otherlibs/stdune/src/temp.ml

```diff
@@ -89,6 +93,16 @@ let create ?perms what ~prefix ~suffix =
        Perhaps, we should use something like [_build/.temp] instead? *)
     Filename.get_temp_dir_name () |> Path.of_filename_relative_to_initial_cwd
   in
+  if contains_path_sep prefix
+  then
+    Code_error.raise
+      "Temp.create: invalid prefix contains path elements"
```

Comment:
The wording is a bit off. The prefix is invalid _because_ it contains path elements, but the error makes it sound like it is an invalid prefix that also contains path elements. I'd suggest "prefix must not contain path elements". Same for the suffix message as well.